src/tasks/submit.rs

```diff
@@ -53,17 +61,14 @@ pub enum ControlFlow {
 }
 
 /// Submits sidecars in ethereum txns to mainnet ethereum
-#[derive(Debug)]
```

Comment:
there should never be 2 of these, if we're cloning it something is wrong. clone impl ought to be removed